docker/fullnode-x/README.md

```diff
@@ -0,0 +1,17 @@
+
+
+1. bring up postgres first so you can create the DB with the `diesel` commands:
+
+- `docker compose up postgres -d`
+- psql -U postgres -p 5432 -h localhost -c 'create database sui_indexer_testnet'
+- run this in sui.git/crates/sui-indexer:
+  * diesel setup --database-url=postgres://postgres:admin@localhost:5432/sui_indexer_testnet
+  * diesel migration run  --database-url=postgres://postgres:admin@localhost:5432/sui_indexer_testnet
+
+
+2. for whichever network you want to use, get the fullnode.yaml config and the genesis.blob files and place them in fullnode/config/
```

Comment:
Do we want to point them to https://docs.sui.io/build/fullnode#building-from-source?

Reply:
ah. right.  I'll add the note about running `docker compose build fullnode` and for indexer. have to build those local to the OS.